### ISSUE TITLE
The annotations on our diagrams are inconsistent.

### ISSUE THREAD

**Issue by joshwooding** [jpmorganchase/salt-ds#3362]
### Package name(s)

Website / Documentation

### Description

- Some annotations include colons after a type e.g. Component: some don't
- Some stack layout annotations, call them a component other's don't
- same type labels are label small, other's aren't.

### Operating system

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

### Are you a JPMorgan Chase & Co. employee?

- [X] I am an employee of JPMorgan Chase & Co.

**Comment by yunjungyeh** [comment]
Work with @ivan-calderon 
- updated the Annotation/Text component in Salt Spec Utilities library 
- added text annotation guides for pattern pages construction

**Comment by yunjungyeh** [comment]
The current update helps new pattern pages be consistent. 
The images in the existing pattern pages would need to be updated when the time is right :) (e.g. update all images when Salt 2 is stable?) cc @joshwooding  @ivan-calderon  @bhoppers2008

**Comment by joshwooding** [comment]
> The current update helps new pattern pages be consistent. 
> 
> The images in the existing pattern pages would need to be updated when the time is right (e.g. update all images when Salt 2 is stable?) cc @joshwooding  @ivan-calderon  @bhoppers2008 

Yeah, we can fix the other images when we do the image updates for the new theme work.
